rclrs/src/rcl_utils.rs

```diff
@@ -0,0 +1,104 @@
+use crate::rcl_bindings::*;
+use crate::{RclrsError, ToResult};
+
+use crate::RclrsError::IndexOutOfRange;
+use libc::c_void;
+use std::ptr::null_mut;
+
+/// Trait used for obtaining information from rcl_arguments_t.
+pub(crate) trait RclGetArg {
```

Comment:
A trait seems to be a little much abstraction. With two function pointers as arguments to `get_rcl_arguments()`, the code would be shorter and in my opinion simpler.

Reply:
I wanted to provide some kind of ready sets of functions to use those `rcl_argument_` API. This way seemed to be the only one. I know it may be an overkill, but I'm still learning this language and treated this like a kind of exercise.